### DISCUSSION TITLE
Trigger a build by a github comment on a pull request

### DISCUSSION THREAD

**Discussion by VincentAP** [codemagic-ci-cd/codemagic-docs#1996] [Ideas]
### Use case

```Text
I would love to be able to trigger a workflow directly through a github's pull request by adding a comment. e.g. I add a comment "build" to the pull request, then a specific workflow that has been defined in the codemagic.yaml file will be triggered.
```

### Idea

```Text
Is it possible to tigger a workflow by commenting on a pull request in Github?
```

**Comment by kulakarla** [comment]
Hello @VincentAP 

This is not possible, since Codemagic is currently not listening to `pull_request_review_comment` Webhook events.

**Reply by VincentAP** [reply]
Will there be any future updates that could add this kind of functionality @kulakarla ?

**Reply by kulakarla** [reply]
In the future - maybe. Currently however, I'm unable to provide you with any kind of confirmation for it.

You can get creative with our current tools. For example, perhaps a pseudo-commit with a specific commit message that would trigger the build. E.g instead of the comment to the pull request, you'll change one comment in some code file and commit it with a `build` commit message. Take a quick peek at our docs for automatic build triggering [here](https://docs.codemagic.io/yaml-running-builds/starting-builds-automatically/), you may find a suitable solution there.

**Reply by VincentAP** [reply]
Noted, thank you.
